### PR TITLE
Add Starship

### LINKED ISSUE #4488: Starship
Name: Starship
Website: starship.rs
Alexa rank: #769,216
Official resources for icon and color:  #b30bb0 (https://github.com/starship/starship)

Starship is a Cross-shell command prompt. Is open-source and has a decent amount contributors on github repository. Also has 13.2k stars.

### PR DESCRIPTION
**Pull request by klu17**
**Issue:** Closes #4488
**Name:** Starship
**Website:** starship.rs
**Alexa rank:** 769,216

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  -  **Why did you pick the hex value?:** @skywarth mentioned `#B30BB0` and `#DD0B78` was used on the website. It appears that both colors are used in the gradient. I went with the last one mentioned in the issue: `#DD0B78`.
  - I manually combined the paths with Inkscape. Then, I ran it through SVGOMG

### Preview
image

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @klu17! 🎉
Just taking a quick look at it, it appears there are some slight differences between your SVG (red) and source (black). They could be as a result of the optimization process, but could you take another look - and optimize to 4 decimals if need be?

image

File: icons/starship.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...63-.1598-1.8346-.1818-.7649-.5334-1.549-.895-1.9955-.0763-.0943-.1387-.2064-.1387-.2493 0-.0428.0564-.2432.1254-.4453.2257-.6618.289-.9892.3134-1.6216.0345-.8923-.0764-1.4378-.4564-2.244-.323-.685-1.1565-1.9088-1.3002-1.9088-.16 0-1.3076.937-2.0371 1.6635-1.052 1.0474-1.5241 1.9453-1.611 3.0634l-.0298.3836-.5955.3266c-1.0605.5816-1.6478 1.0097-2.1986 1.6026-.1706.1836-.3102.3465-.3102.3619 0 .0234 2.246.9546 2.6004 1.0782.0624.0217.113.07.1128.1074-.0002.0373-.065.298-.1439.5792-.0789.2812-.1434.542-.1434.5796 0 .0774-.0422.0634 1.9119.6354 1.6914.495 1.5936.4681 1.6652.46.0384-.0045.1425-.2092.2752-.5416zM6.76 7.718c.2105-.469.701-.8226 1.4156-1.0207.4156-.1151.4155-.1145.0178-.2069-.4944-.115-.9934-.4566-1.2466-.8537-.1381-.2167-.2169-.4061-.3176-.7638-.0371-.1318-.0763-.2307-.0871-.2199-.0108.0108-.0476.1278-.0819.26-.1655.6394-.5555 1.1511-1.0578 1.3879-.161.0759-.4205.1714-.5766.2123l-.2838.0744.272.0918c.8514.2872 1.3598.7808 1.6163 1.5693l.1142.3509.0543-.3217c.0299-.177.1024-.429.1612-.56zm7.228 2.8267c-.6619-.3082-.7901-1.226-.2389-1.71.4425-.3885 1.0264-.3667 1.435.0537.227.2334.3078.4311.3072.7516-.0008.3781-.2851.7966-.6429.946-.2171.0908-.618.0715-.8604-.0413z"/></svg>
```
Comment (adamrusted) [inline icons/starship.svg:1]: This version is still showing some differences... :thinking:

I'll take a look and see if I'm able to get a path made up for this...

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
Could another @simple-icons/maintainers take a look at this? The path I've included should be fine - but would be good to get your input before merging.

**Comment by mondeja** [comment]
I'm able to optimize with 3 decimals and without seeing differences against the original resized to 24px. This is the path I got:

```svg
M12 0A12 12 0 0 0 0 12a12 12 0 0 0 4.684 9.512c4.1-.44 5.598-3.67 5.598-3.67l3.226 1.053s-.53 3.17-2.547 5.06A12 12 0 0 0 12 24a12 12 0 0 0 12-12A12 12 0 0 0 12 0zM6.597 4.575c.146 1.2.8 1.9 2.03 2.03-1.364.208-2.05.88-2.03 2.033-.2-1.114-.863-1.805-2.03-2.032 1.26-.15 1.89-.872 2.03-2.03zm8.584.992c1.626 1.994 2.345 3.763 1.234 6.246.92.997 1.314 2.598 1.3 4.094l-2.825-.65-.444 1.18-3.637-1.062.367-1.267-2.78-1.103c.887-1.083 1.773-1.615 3.148-2.29-.123-2.125 1.593-3.648 3.636-5.148zm-.716 2.995c-.584 0-1.057.474-1.057 1.058 0 .584.473 1.057 1.057 1.057s1.057-.473 1.057-1.057-.472-1.058-1.056-1.058z
```

Could you check it @adamrusted?

**Comment by adamrusted** [comment]
> I'm able to optimize with 3 decimals and without seeing differences against the original resized to 24px. This is the path I got:
> 
> ```svg
> M12 0A12 12 0 0 0 0 12a12 12 0 0 0 4.684 9.512c4.1-.44 5.598-3.67 5.598-3.67l3.226 1.053s-.53 3.17-2.547 5.06A12 12 0 0 0 12 24a12 12 0 0 0 12-12A12 12 0 0 0 12 0zM6.597 4.575c.146 1.2.8 1.9 2.03 2.03-1.364.208-2.05.88-2.03 2.033-.2-1.114-.863-1.805-2.03-2.032 1.26-.15 1.89-.872 2.03-2.03zm8.584.992c1.626 1.994 2.345 3.763 1.234 6.246.92.997 1.314 2.598 1.3 4.094l-2.825-.65-.444 1.18-3.637-1.062.367-1.267-2.78-1.103c.887-1.083 1.773-1.615 3.148-2.29-.123-2.125 1.593-3.648 3.636-5.148zm-.716 2.995c-.584 0-1.057.474-1.057 1.058 0 .584.473 1.057 1.057 1.057s1.057-.473 1.057-1.057-.472-1.058-1.056-1.058z
> ```
> 
> Could you check it @adamrusted?

I've pushed a commit with a 3 decimal path above, could you check it and merge if it's good @mondeja?

**Review by mondeja — APPROVED** [review]
>I've pushed a commit with a 3 decimal path above, could you check it and merge if it's good @mondeja?

Oh, I didn't see the complete commit message :man_facepalming: Yes, It's perfect!
Thank you @adamrusted and @klu17 for this contribution! :pray:

**Pull request merged**